src/components/calcite-tile-select/calcite-tile-select.tsx

```diff
@@ -141,6 +164,9 @@ export class CalciteTileSelect {
 
   connectedCallback(): void {
     this.renderInput();
+    if (this.name) {
+      this.checkFirstTileSelect();
```

Comment:
Should this also be caled if `name` is changed? Maybe it needs to watch that prop?

Reply:
I can't imagine a real world use case for a radio group's name changing, but I could certainly be proven wrong.  At any rate, this function is really only meant to run on the initial component setup, because once the component is mounted and interactable, the normal checking/unchecking behavior that then occurs from user interaction, any set of radios that simultaneously have their `checked` property initially set will get reset from the callbacks setup to respond to the native inputs that they render.  Does that make sense?

Reply:
You never know. The name of a radio could change depending on another form value. "name" is an attribute so it can be changed at any time.

Reply:
Also, I'm not sure this has any effect, but I believe `connectedCallback` is called anytime a node is moved or appended to another node. So if someone's JSX has this node moving places in the DOM this would be called again.

Reply:
Looking at the code, it seems like if the `name` prop is modified it should modify the `input` internally.

Reply:
I'll try setting up a test case to see what happens when the `name` is programatically changed.  That will give me a better sense of what the behavior should be.  I might also try doing this with the native radio input to compare.  Will report back.